website/docs/r/user_provided_service.html.markdown

```diff
@@ -1,6 +1,6 @@
 ---
-layout: "cf"
-page_title: "Cloud Foundry: cf_user_provided_service"
+layout: "cloudfoundry"
+page_title: "Cloud Foundry: cloudfoundry_user_provided_service"
 sidebar_current: "docs-cf-resource-user-provided-service"
```

Comment:
I assume the sidebar_current ids are defined in [cf.erb](https://github.com/mevansam/terraform-provider-cf/blob/dev/website/cf.erb) and therefore don't need renames

Reply:
For backward compatibility would it make sense to register providers under the cf_* name a well as cloudfoundry_*, until Janos has moved over to the new naming convention? Theoretically, it should work but I have not tested it but wanted to validate if this has already been experimented with at Orange.

Reply:
@mevansam I would suggest to keep the provider code simple and compliant to hashicorp guidelines, and rather rely on beta users of mevansam/terraform-provider-cf to upgrade themselves using the procedure @janosbinder kindly contributed.

Reply:
I do agree with @gberche-orange this is more complexity for us to maintain than for user to do the changes. In add, we will never remove it to ensure compatibility by doing. I think that the sooner the better here.